backend/models/user.py

```
@@ -1,5 +1,6 @@
 from sqlalchemy import Column, Integer, String, Boolean
 from database.session import Base
+from sqlalchemy.orm import relationship
```

Ruff (0.8.2)

`sqlalchemy.orm.relationship` imported but unused

Remove unused import: `sqlalchemy.orm.relationship`

(F401)